core/src/style/pointStyleBuilder.cpp

```diff
@@ -275,38 +270,62 @@ void PointStyleBuilder::addLabel(const Point& _point, const glm::vec4& _quad,
         });
 }
 
-bool PointStyleBuilder::getUVQuad(PointStyle::Parameters& _params, glm::vec4& _quad) const {
+bool PointStyleBuilder::getUVQuad(PointStyle::Parameters& _params,
+                                  glm::vec4& _quad, Texture** _texture) const {
+
     _quad = glm::vec4(0.0, 1.0, 1.0, 0.0);
+    auto texture = m_style.defaultTexture().get();
 
-    if (m_style.spriteAtlas()) {
-        SpriteNode spriteNode;
+    if (!_params.texture.empty()) {
 
-        if (!m_style.spriteAtlas()->getSpriteNode(_params.sprite, spriteNode) &&
-            !m_style.spriteAtlas()->getSpriteNode(_params.spriteDefault, spriteNode)) {
-            return false;
-        }
+        if (_params.texture == "null") {
+            texture = nullptr;
+        } else {
+            if (!m_style.textures()) {
```

Comment:
I am not sure this is required, this check is mandating that the style must define a texture for it to be overriden in the draw group. Based on the description of the problem in #1645 that does not seem to be case. Can you clarify?

Reply:
This is just a nullptr check before accessing the global texture map :)